dbms/src/Common/TiFlashMetrics.h

```diff
@@ -178,10 +178,13 @@ namespace DB
     M(tiflash_storage_page_write_duration_seconds, "The duration of each write batch", Histogram,                                         \
         F(type_total, {{"type", "total"}}, ExpBuckets{0.0001, 2, 20}),                                                                    \
         F(type_blob,  {{"type", "blob"}},  ExpBuckets{0.0001, 2, 20}),                                                                    \
```

Comment:
We can remove this since we break it down to `choose_stat`/`search_pos`/`blob_write` in this PR